### ISSUE TITLE
[request] add SVG icon

### ISSUE THREAD

**Issue by edbefee3-3888-462a-9411-741b7e9eb54e** [Arisotura/melonDS#1168]
i feel like the name is self-explanatory

**Comment by poudink** [comment]
the original icon isn't svg https://github.com/Arisotura/melonDS/issues/202#issuecomment-340246213

**Comment by mariomadproductions** [comment]
I think you'd need to contact limmitox

**Comment by poudink** [comment]
I don't think it would help. There was never an SVG. (https://github.com/Arisotura/melonDS/issues/202#issuecomment-341628120) The icon would have to be completely remade.

**Comment by mariomadproductions** [comment]
> I don't think it would help. There was never an SVG. ([#202 (comment)](https://github.com/Arisotura/melonDS/issues/202#issuecomment-341628120)) The icon would have to be completely remade.

yes but the photoshop file, if he still has it, may have usee vector graphics, and therefore could be converted to svg

**Comment by WaluigiWare64** [comment]
Here is my attempted vectorisation of the melonDS logo. You may spot some imperfections but those are because of the original image.
[melon.zip](https://github.com/Arisotura/melonDS/files/6948242/melon.zip)

**Comment by edbefee3-3888-462a-9411-741b7e9eb54e** [comment]
it has some problems (such as the colors being too dark) but it's pretty good
